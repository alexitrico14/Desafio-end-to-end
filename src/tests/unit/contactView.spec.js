import { shallowMount } from '@vue/test-utils';
import ContactView from '@/views/ContactView.vue';

describe('ContactView.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(ContactView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
