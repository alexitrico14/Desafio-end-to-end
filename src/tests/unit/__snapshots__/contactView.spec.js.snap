// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView.vue matches snapshot 1`] = `
"<div class="contact">
  <h1>Contact Us</h1>
  <p>This is the Contact page.</p>
</div>"
`;
